### PR TITLE
grimblast: implement --wait

### LINKED ISSUE #57: grimblast: add --wait option
This would delay taking the screenshot by the given amount of seconds. 

https://github.com/OctopusET/sway-contrib/blob/master/grimshot.1.scd?plain=1#L21-L24

### PR DESCRIPTION
**Pull request by fufexan**
Fixes #57.

Inspired by https://github.com/OctopusET/sway-contrib/commit/695c70deab5279b61df8ec9420030c133599253a.

### CODE REVIEW

**Comment by fufexan** [comment]
@andresilva can you test this and verify it works as intended?

File: grimblast/grimblast [hunk 1]
```diff
@@ -42,6 +42,7 @@ env_editor_confirm() {
 NOTIFY=no
 CURSOR=
 FREEZE=
+WAIT=
```
Comment (andresilva) [inline grimblast/grimblast:45]: Shouldn't this be initialized to "no"?

**Review by andresilva — CHANGES_REQUESTED** [review]
Haven't tested yet.

File: grimblast/grimblast (hunk 1 above)
Comment (fufexan) [reply grimblast/grimblast:45]: Yes, you're right.

**Review by andresilva — APPROVED** [review]
Works as expected.

**Pull request merged**